### PR TITLE
update documentation

### PR DESCRIPTION
**Pull request by kat-hi**
I updated the documentation according to our discussions.
- add a JWT to the login response
- require request header "Authorization":  eyhfinerofmer.fwiogklf.fwnfiowew for api calls, except /login

- I already implemented these changes in kubeportal: https://github.com/kubeportal/kubeportal/pull/94

please change the kubeportal-api accordingly and provide api-access if the Authorization header (with a correct token) is given

### CODE REVIEW

**Review by troeger — APPROVED** [review]
Looks all fine.

**Comment by kat-hi** [comment]
does this documentation fit to your kubeportal-django logic? please update this documentation according to your api-truth

File: documentation/openapi.yaml [hunk 1]
```diff
@@ -274,6 +296,9 @@ components:
         firstname: 
           type: string
           example: 'Mandarin'
+        token:
+          type: string
+          example: 'eyfjopgfvdvfer.fpkksdner.jhjdemvpdf'
```
Comment (troeger) [inline documentation/openapi.yaml:301]: A correct example would be:

```
Bearer eyfjopgfvdvfer
```

**Review by troeger — APPROVED** [review]
Only the JWT example is wrong, beside that, eveything is fine.

**Comment by kat-hi** [comment]
..and I'll add csrf token in LoginSuccess payload.

**Pull request merged**